Simonyi/index.rst

```diff
@@ -41,8 +41,7 @@ Introduction to the Simonyi Telescope
 
    Components/MTCS/index.rst
    Components/*
-   Components/Auxiliary-Components/index.rst
-
+   
 .. _Simonyi-Telescope-Weather-Constraints:
 
 Weather Constraints
```

Comment:
There are 2 links that take to the same page "Simonyi Weather Constraints"

Reply:
Thank you, I have add the option `:hidden:` to the `toctree` so it shows in the left hand side menu, but not in the page itself.